.eslintrc.js

```diff
@@ -10,11 +12,16 @@ module.exports =  {
   },
   plugins: [
     '@typescript-eslint',
+    'chai-friendly',
   ],
   rules: {
     // note you must disable the base rule as it can report incorrect errors
     '@typescript-eslint/indent': ['error', 2],
     '@typescript-eslint/explicit-function-return-type': ['off'],
     '@typescript-eslint/no-explicit-any': ['off'],
+    '@typescript-eslint/no-unused-expressions': 0,
```

Comment:
what is the meaning of 0 and 2 ?
Looks like the official options are bit different ... https://eslint.org/docs/rules/no-unused-expressions ?

Reply:
thats because we use chai as framework and we're using the "chai-friendly". 0 disables the rule and 2 enables the chai-friendly rule

https://www.npmjs.com/package/eslint-plugin-chai-friendly

Reply:
ah my point was just about using numbers instead of strings like "off" or "error" bc we mostly used those in the config and I had the feeling using numbers 0,1,2 was beeing deprecated. Seems like it's still supported fully, but maybe we should keep to our config style despite that.

https://eslint.org/docs/user-guide/configuring#configuring-rules

Reply:
yes, but shouldn't block this particular PR, since it's general discussion on code styles, maybe we should move the conversation to the chat.